### ISSUE TITLE
Remember feature and Strong Session

### ISSUE THREAD

**Issue by ar-anvd** [miguelgrinberg/flasky#178]
As far as I can see [here](https://github.com/maxcountryman/flask-login/issues/248), the remember feature is not compatible with strong session. Also, behavior confirmed in my projects. I need to use basic instead.

https://github.com/miguelgrinberg/flasky/blob/master/app/auth/views.py#L35
https://github.com/miguelgrinberg/flasky/blob/9bcab0e17504629b4bfe7ca557f1fc44d920f767/app/__init__.py#L17

**Comment by miguelgrinberg** [comment]
This is unfortunate but correct.

The version of Flask-Login that I used back when I wrote the application for the book was 0.2.9, and in this release the use of strong session protection along with the remember me cookie worked, by having the remember me setting take precedence over the strong session protection check when present. That functionality broke in later versions of Flask-Login, and the author believes strong sessions and remember me should not be used together anyway, so he has not accepted the request to restore this functionality to the way it was in 0.2.9.

I guess I should change the strong session setting to basic in the code, to avoid any confusions.
